### PR TITLE
CONCF-589 Wikia Home Sponsor Images

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/CONCF-589

Fix sizes of sponsor images to render and display correctly. Details in the ticket.

ping @Warkot @nandy-andy 

Tests are failing due to other, not related issues.

### CODE REVIEW

File: extensions/wikia/WikiaHomePage/js/WikiaHomePage.js [hunk 1]
```diff
@@ -535,8 +543,8 @@ WikiaHomePageRemix.prototype = {
 		if( this.heroImage !== null ) {
 			img = $(this.heroImage)
 				.attr('alt', imgData.title)
-				.attr('witdh', imgData.width)
-				.attr('witdh', imgData.height);
+				.attr('witdh', this.SPONSOR_HERO_IMAGE_WIDTH)
```
Comment (rogatty) [inline extensions/wikia/WikiaHomePage/js/WikiaHomePage.js]: Typo.

Comment (dianafa) [inline extensions/wikia/WikiaHomePage/js/WikiaHomePage.js]: Where? It actually WAS typo here. If you mean double `width` here: `.attr('witdh', imgData.width).attr('witdh', imgData.height);`

Comment (Warkot) [inline extensions/wikia/WikiaHomePage/js/WikiaHomePage.js]: `.attr('wi**td**h'…` td → dt

Comment (dianafa) [inline extensions/wikia/WikiaHomePage/js/WikiaHomePage.js]: wow... Thanks!

**Comment by rogatty** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM :shipit:

**Pull request merged**